packages/cookie-handler/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@goldinteractive/feature-cookie-disclaimer",
```

Comment:
While I'm fine with the new name `cookie-handler`, this currently contains the wrong package name. (Also see "description")